Lesson013/HW013.MotorcyclesStore/MotorcyclesStore/Controllers/MotoStoreController.cs

```diff
@@ -0,0 +1,127 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using MotorcyclesStore.Core.Interfaces;
+using MotorcyclesStore.Core.Models;
+using Serilog;
+
+namespace MotorcyclesStore.UI.Controllers
+{
+    public class MotoStoreController : IMotoStoreController
+    {
+        private readonly IMotoStoreService _motoStoreService;
+
+        public MotoStoreController(IMotoStoreService motoStoreServicevice)
+        {
+            _motoStoreService = motoStoreServicevice;
+        }
+
+        public void GetAll()
+        {
+            var allMoto = _motoStoreService.GetMotorcycles();
+            Console.ForegroundColor = ConsoleColor.Cyan;
+            Console.WriteLine(allMoto);
+            Console.ResetColor();
+        }
+
+        public void GetFirstOrDefaultById(int Id)
+        {
+            Log.Logger.Information("Method GetFirstOrDefaultById STARTED");
+
+            try
+            {
+                Console.ForegroundColor = ConsoleColor.Red;
+                Console.WriteLine("Enter Motorcycle ID: ");
+                var userInputId = Convert.ToInt32(Console.ReadLine());
+                Log.Logger.Debug("Entering the moto ID", userInputId);
+                var motoInfo = _motoStoreService.GetMotorcycleByID(userInputId);
+                Log.Logger.Information(" Get Info Motorcycle");
+                Console.ForegroundColor = ConsoleColor.Cyan;
+                Console.WriteLine($"Id: {motoInfo.Id}  Name: {motoInfo.Name}  Model: {motoInfo.Model}  Year: {motoInfo.Year}  Odometer: {motoInfo.Odometer}  Price: {motoInfo.Price}\n");
+                Console.ResetColor();
```

Comment:
You create an info line here, and for the method GetAllMoto, you do it in the service. Choose one place for this. or create a separate method in the service.